guides/common/modules/proc_importing-into-the-library-environment-from-a-web-server.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Importing_into_the_Library_Environment_from_a_web_server_{context}"]
+= Importing into the Library Environment from a web server
+
+You can import exported Library content directly from a web server into the Library lifecycle environment of an organization on another {ProjectServer}.
+For more information about exporting contents from the Library environment, see xref:Exporting_the_Library_Environment_{context}[].
+
+.Prerequisites
+* The exported files must be in a syncable format.
+* The exported files must be accessible via HTTP/HTTPS.
```

Comment:
same as above.

I think the general rule is to not use latin words like "via".